### PR TITLE
feat: add worst in input option to  response time timeseries

### PR DESCRIPTION
**Pull request by taro-yu**
## Description
This PR adds worst_in_input case option to the API: `create_response_time_timeseries_plot()`.

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been properly tested.
- [x] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] (Optional) The PR has been properly tested with CARET_report verification.
- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: src/caret_analyze/plot/plot_facade.py [hunk 1]
```diff
@@ -188,7 +188,7 @@ def create_response_time_timeseries_plot(
             This also accepts multiple inputs by unpacking.
         case: str, optional
             Response time calculation method, best by default.
-            supported case: [best/worst/all].
+            supported case: [best/worst/worst_in_input/all].
```
Comment (rokamu623) [inline src/caret_analyze/plot/plot_facade.py:191]:
```suggestion
            supported case: [best/worst/worst-in-input/all].
```

File: src/caret_analyze/plot/timeseries/response_time_timeseries.py [hunk 2]
```diff
@@ -111,5 +111,7 @@ def to_timeseries_records_list(
                 response_timeseries_list.append(response.to_worst_case_records())
             elif self._case == 'all':
                 response_timeseries_list.append(response.to_all_records())
+            elif self._case == 'worst_in_input':
```
Comment (rokamu623) [inline src/caret_analyze/plot/timeseries/response_time_timeseries.py:114]:
```suggestion
            elif self._case == 'worst-in-input':
```

File: src/caret_analyze/plot/timeseries/timeseries_plot_factory.py [hunk 3]
```diff
@@ -54,7 +54,8 @@ def create_instance(
         visualize_lib : VisualizeLibInterface
             Instance of VisualizeLibInterface used for visualization.
         case : str
-            Parameter specifying best, worst or all. Use to create Response time timeseries graph.
+            Parameter specifying best, worst, worst_in_input, or all.
```
Comment (rokamu623) [inline src/caret_analyze/plot/timeseries/timeseries_plot_factory.py:57]:
```suggestion
            Parameter specifying best, worst, worst-in-input, or all.
```

File: src/caret_analyze/plot/visualize_lib/bokeh/bokeh.py [hunk 4]
```diff
@@ -152,7 +152,8 @@ def timeseries(
             If True, all legends are drawn
             even if the number of legends exceeds the threshold.
         case : str
-            Parameter specifying best, worst or all. Use to create Response time timeseries graph.
+            Parameter specifying best, worst, worst_in_input, or all.
```
Comment (rokamu623) [inline src/caret_analyze/plot/visualize_lib/bokeh/bokeh.py:155]:
```suggestion
            Parameter specifying best, worst, worst-in-input, or all.
```

**Review by rokamu623 — CHANGES_REQUESTED** [review]
Please replace `_` to `-` in strings.

File: src/caret_analyze/plot/plot_facade.py (hunk 1 above)
Comment (taro-yu) [reply src/caret_analyze/plot/plot_facade.py:191]: Fixed in https://github.com/tier4/CARET_analyze/pull/336/commits/318c038007dfb0f14bd6648fe8dedc6824680d2a.

File: src/caret_analyze/plot/timeseries/response_time_timeseries.py (hunk 2 above)
Comment (taro-yu) [reply src/caret_analyze/plot/timeseries/response_time_timeseries.py]: Fixed in https://github.com/tier4/CARET_analyze/pull/336/commits/318c038007dfb0f14bd6648fe8dedc6824680d2a.

File: src/caret_analyze/plot/timeseries/timeseries_plot_factory.py (hunk 3 above)
Comment (taro-yu) [reply src/caret_analyze/plot/timeseries/timeseries_plot_factory.py:57]: Fixed in https://github.com/tier4/CARET_analyze/pull/336/commits/318c038007dfb0f14bd6648fe8dedc6824680d2a.

File: src/caret_analyze/plot/visualize_lib/bokeh/bokeh.py (hunk 4 above)
Comment (taro-yu) [reply src/caret_analyze/plot/visualize_lib/bokeh/bokeh.py]: Fixed in https://github.com/tier4/CARET_analyze/pull/336/commits/318c038007dfb0f14bd6648fe8dedc6824680d2a.

File: src/caret_analyze/plot/plot_facade.py (hunk 1 above)
Comment (rokamu623) [reply src/caret_analyze/plot/plot_facade.py:191]: Not fixed.

File: src/caret_analyze/plot/timeseries/timeseries_plot_factory.py (hunk 3 above)
Comment (rokamu623) [reply src/caret_analyze/plot/timeseries/timeseries_plot_factory.py:57]: Not fixed.

**Review by rokamu623 — CHANGES_REQUESTED** [review]

**Review by rokamu623 — DISMISSED** [review]
LGTM!

**Review by nabetetsu — DISMISSED** [review]
LGTM :+1:
Please check pytest error in this PR.

rt2-823 drawio

**Review by rokamu623 — APPROVED** [review]
LGTM

**Pull request merged**